ocs_ci/ocs/constants.py

```diff
@@ -522,6 +523,11 @@
     '4.2.0': OCP4_2_REPO
 }
 
+# UPI script paths
```

Comment:
Do these need to be in constants since they are only used in AWSUPI?

Reply:
VMware also uses same constant.

Reply:
oh sorry, you meant line 527 - 529 ? that's not required